cmake/Modules/EttercapLibCheck.cmake

```diff
@@ -117,22 +117,18 @@ endif(HAVE_ICONV)
 
 # LTDL
```

Comment:
I would rather say something like:
```
if(ENABLE_PLUGINS)
  set(HAVE_PLUGINS 1)
  set(EC_LIBS ${EC_LIBS} ${CMAKE_DL_LIBS})
  set(EC_LIBETTERCAP_LIBS ${EC_LIBETTERCAP_LIBS} ${CMAKE_DL_LIBS})
endif(ENABLE_PLUGINS)
```

the variables are empty if not found, and I guess cmake is able to find CMAKE_DL_LIBS also in dlopen.

So I guess everytime is found and goes in the first branch, hence we can drop it :)
what is your opinion?